File: LC-001/java/Solution.java
public class Solution {
    public static void main(String[] args){
        int[] nums= new int[]{2, 3, 4};
        int target = 7;

        int[] res = solution(nums, target);

        System.out.println("result is " + res[0] + " and " + res[1]);
    }

    public static int[] solution(int[] nums, int target){
        for(int i=1; i<nums.length; i++){
            for(int j=0; j<i; j++){
                if((nums[i] + nums[j]) == target){
                    return new int[]{j, i};
                }
            }
        }

        return new int[]{-1, -1};
    }
}

File: LC-001/kotlin/Solution.kt
fun main(args: Array<String>) {
    val nums = arrayOf(2, 3, 4)
    val target = 7

    val res = solution(nums, target)

    println("result is ${res[0]}, and ${res[1]}")
}

fun solution(nums: Array<Int>, target: Int): Array<Int> {
    for (i in 1..nums.size) {
        for (j in 0..i) {
            if ((nums[i] + nums[j]) == target) {
                return arrayOf(j, i)
            }
        }
    }

    return arrayOf(-1, -1)
}
